code/game/objects/machinery/computer/camera_advanced.dm

```diff
@@ -158,6 +160,55 @@
 	eyeobj.setLoc(eyeobj.loc)
 
 
+/obj/machinery/computer/camera_advanced/proc/track(mob/living/target)
+	if(!istype(target))
+		return
+
+	INVOKE_ASYNC(src, .track_actual, target)
```

Comment:
```suggestion
	INVOKE_ASYNC(src, .proc/track_actual, target)
```

Reply:
Or just call it regularly. `INVOKE_ASYNC` is to make it `set waitfor = FALSE`, but this is already done in the proc.